### DISCUSSION TITLE
How to change the color when drag the block to the canvas

### DISCUSSION THREAD

**Discussion by mingxin-yang** [GrapesJS/grapesjs#3946] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/23076937/141748836-9b1af559-7740-4da2-9398-5424cd3d62cf.png)

the yellow color

**Comment by mingxin-yang** [comment, marked as answer]
.gjs-selected-parent {
                    outline:2px solid transparent  !important;
                  }

**Comment by abhishekSolulab** [comment]
> ![image](https://user-images.githubusercontent.com/23076937/141748836-9b1af559-7740-4da2-9398-5424cd3d62cf.png)
> 
> The green color

if we want to change that green line color then what could be the possibilities and how we can do that?

**Reply by abhishekSolulab** [reply]
i got the answer for the same .
we need to replace CSS with this CSS for change color
.gjs-placeholder.horizontal, .gjs-com-placeholder.horizontal, .gjs-placeholder.horizontal {
  border-color: transparent #363636b0 !important;
    // border-width: 3px 5px;
    // margin: -3px 0 0;
}
.gjs-placeholder-int, .gjs-com-placeholder-int, .gjs-placeholder-int{
  background-color: #363636b0 !important;
}
